devel/protobuf3-cpp/Portfile

```diff
@@ -52,6 +46,7 @@ checksums       rmd160  f7c2879e427160c2a5055a2ab5d5ab231304fd31 \
 
 github.tarball_from releases
 distname        protobuf-cpp-${version}
+dist_subdir     protobuf3-cpp
```

Comment:
Same: For the various protobuf* ports, `dist_subdir` should be set to `protobuf-cpp`.